libraries/lambda_handlers/run_linter_handler.py

```diff
@@ -18,13 +18,16 @@ def _handle(self, event, context):
         commit_data = self.retrieve(self.data, 'commit_data', 'payload', required=False)
         resource_id = self.retrieve(self.data, 'resource_id', 'payload', required=False)
         single_file = self.retrieve(self.data, 'single_file', 'payload', required=False, default=None)
+        lint_callback = self.retrieve(self.data, 'lint_callback', 'payload', required=False, default=None)
+        identity = self.retrieve(self.data, 'identity', 'payload', required=False, default=None)
 
         # Execute
         linter_class = LinterHandler.get_linter_class(resource_id)
         linter = linter_class(source_zip_url=source_zip_url, commit_data=commit_data, resource_id=resource_id,
-                         single_file=single_file)
+                              single_file=single_file, lint_callback=lint_callback, identity=identity)
         ret_value = linter.run()
-        if App.linter_messaging_name:
+        if App.linter_messaging_name:  # TODO this shouldn't have been hard coded, now we are sending lint messages
```

Comment:
Ah, it isn't necessarily hard coded as it could be replaced by a stage variable, but I see you're saying we are sending lint messages even for non multi-part linting jobs? Best way to update this would be in the client_webhook to set `App.linter_message_name = None` if not multi.

Reply:
Not really sure how this is any more hard-coded then it was coming from a Stage Variable. How did we turn off linter messaging before I mucked with it?

Reply:
we sent the messaging name in the payload, but only for multi-part.

Reply:
Ah, right, it didn't come from the stage variables for the linter since we invoked.  Well, we could still send it as "null" in the payload and that should set it to None in the linter.

Reply:
All App hardcoded values get clobbered by anything in event['vars']

Reply:
Or we could make the default value be null and let the client_webhook pass a messenger name it looks for.

Reply:
That would be better.